packages/dragdrop/src/index.ts

```diff
@@ -1232,6 +1232,7 @@ namespace Private {
         ? doc.body
         : (doc.firstElementChild as HTMLElement);
     if (!cursorBackdrop.isConnected) {
+      cursorBackdrop.style.transform = 'scale(0)';
```

Comment:
This is a clever reuse of transform which will get overriden on mousemove. Can you add a comment explaining what you described in the last post here?

Reply:
Good point! Comment added.

Reply:
The only thing I'd add is that it's still not obvious to me what the mechanism by which this transform is (if it is) overwritten on mouse move by this comment.